internal/artifact/artifact.go

```diff
@@ -54,6 +54,8 @@ const (
 	BrewTap
 	// GoFishRig is an uploadable Rigs rig food file.
 	GoFishRig
+	// KrewPlugin is a krew plugin file.
+	KrewPlugin
```

Comment:
IMHO, i would call it only krew?

Reply:
I have no idea... I think the correct would be `KrewManifest` 🤔  (judging by `validate-krew-manifest`)... will rename :)